test/SlowTests/Corax/RavenDB-21642.cs

```diff
@@ -0,0 +1,68 @@
+#nullable enable
+using System;
+using System.Linq;
+using FastTests;
+using Raven.Client.Documents.Indexes;
+using Raven.Client.Documents.Linq;
+using Tests.Infrastructure;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace SlowTests.Corax;
+
+public class RavenDB_21642 : RavenTestBase
+{
+    public RavenDB_21642(ITestOutputHelper output) : base(output)
+    {
+    }
+    
+    [RavenFact(RavenTestCategory.Querying | RavenTestCategory.Corax)]
+    public void CanUseInAndAllInWithOnlyOneElement()
+    {
```

Comment:
No reason why this test cannot be executed in both Lucene and Corax.

Reply:
Agree, changed into theory.